Updates to documentation of models

KalervoHyyppa opened the pull request:
## Description

Adds additional documentation for the modes used within Ribn.

Also added documentation to Confluence: https://topl.atlassian.net/wiki/spaces/UR/pages/505806874/Models

Fixes # (issue)

Dion/Ribn Data Models #161

## Type of change

-   [x] Documentation update

## How Has This Been Tested?

NA

## Checklist:

-   [x] I have performed a self-review of my own code
-   [x] I have commented my code, particularly in hard-to-understand areas
-   [x] I have made corresponding changes to the documentation
-   [x] My changes generate no new warnings

mgrand-topl reviewed (commented):
You have done a good job of documenting the fields of some classes. Please also document how the classes are used. When are they created? Are they populated from the UI or from the back end?

KalervoHyyppa commented:
I've updated and added more models to [Confluence](https://topl.atlassian.net/wiki/spaces/UR/pages/505806874/Models) with details on their usage and creation.

ultimaterex reviewed (approved):
Thanks for adding some much needed documentation 🐙
Approved
